internal/services/git/git.go

```diff
@@ -144,14 +148,8 @@ func (g *Git) parseLineStringToNumber(line string) string {
 	return strconv.Itoa(num)
 }
 
-func (g *Git) getCleanOutput(output []byte) []byte {
-	// Output from git log contains the diff changes
-	// so we need to extract only the json output data.
-	if idx := bytes.LastIndex(output, []byte("}")); idx >= 0 {
-		return bytes.ReplaceAll(output[:idx+1], []byte("^^^^^"), []byte(`"`))
-	}
-	logger.LogWarn(fmt.Sprintf("Could not to clean git blame output: %s", output))
-	return []byte("")
+func (g *Git) replaceCarets(output []byte) []byte {
```

Comment:
I took a closer look at the code and saw this function with some unnecessary stuff. Changed to just relace the carets.